src/app/components/apparatus-entry/apparatus-entry-readings/apparatus-entry-readings.component.html

```diff
@@ -10,7 +10,7 @@
     </ng-container>
 
     <ng-container *ngIf="data.lemma.content.length > 0">
-        <span *ngFor="let witID of data.lemma.witIDs"> {{ witID }}</span>
+        <span *ngFor="let witID of data.lemma.witIDs" class="appa-witn"> {{ witID }}</span>
```

Comment:
Can you find a better name for this class? Maybe "app-wits"? We use "app" and "wits" in the project